example-deployment/keycloak-tools/bootstrap-keycloak.js

```diff
@@ -152,7 +167,9 @@ async function createAudienceClientScope(headers, name, description, audience) {
       'reporting',
     );
 
-    // get existing clients (dashboard and reporting)
+    await createAudienceClientScope(authHeaders(token), 'minimal', 'minimal scope', 'minimal');
```

Comment:
ocd: could we format this the same as in previous paragraphs :laughing:  

also worth double checking this part works, i think i tried this script once and the audience scope didn't propagate, but I might have been careless, and looking at the script I dont see anything wrong so its probably me.